js/repl/PluginConfig.ts

```diff
@@ -66,7 +66,7 @@ const envPresetDefaults = {
     default: "3.21",
   },
   modules: {
-    default: false,
+    default: "false",
```

Comment:
Q: Why is this a string?

Reply:
![image](https://github.com/user-attachments/assets/dc2dcc5e-2b77-4ab4-998b-15598b4a8335)
`select.value` does not accept `false`.

Thanks for your reminder, I checked the code and it handles the `false` of string.
![image](https://github.com/user-attachments/assets/098b72bb-2c4f-4679-919d-bb24604576f5)

Reply:
No, it seems like it should be done another way. :)